src/server/server.h

```diff
@@ -355,9 +356,9 @@ class Server {
   std::mutex db_job_mu_;
   bool db_compacting_ = false;
   bool is_bgsave_in_progress_ = false;
-  int64_t last_bgsave_time_ = -1;
+  int64_t last_bgsave_time_in_secs_ = -1;
```

Comment:
There are 2 variables with `in` before `secs`. In other places, we don't have `in` as far as I see.

Reply:
Sigh, I think we can keep this, this is because:

```c++
    string_stream << "last_bgsave_time:"
                  << (last_bgsave_time_secs_ == -1 ? start_time_secs_ : last_bgsave_time_secs_) << "\r\n";
    string_stream << "last_bgsave_status:" << last_bgsave_status_ << "\r\n";
    string_stream << "last_bgsave_time_sec:" << last_bgsave_time_spent_secs_ << "\r\n";
```

Origin name has a "last_bgsave_time_sec" and "last_bgsave_time", I just want to separate them

Reply:
@mapleFU  I think you can resolve that confusion by using `last_bgsave_duration_secs_` and `last_bgsave_timestamp_secs` (or even omit `_secs` since the timestamp is in seconds in most cases).

Reply:
Updated